services/traefik/docker-compose.yml.j2

```diff
@@ -90,16 +97,15 @@ services:
         # via https://community.traefik.io/t/v2-2-8-global-redirect-www-to-non-www-with-http-to-https/7428
         # see also: https://community.traefik.io/t/get-a-valid-ssl-certificate-for-www-domains-via-traefik-and-lets-encrypt/2023
         # Global redirection: https (www.) to https
-        - traefik.http.routers.wwwsecure-catchall.rule=HostRegexp(`(?P<host>(www\.).+)`)
-        - traefik.http.routers.wwwsecure-catchall.entrypoints=https
-        - traefik.http.routers.wwwsecure-catchall.tls=true
-        - traefik.http.routers.wwwsecure-catchall.middlewares=wwwtohttps
-        # middleware: http(s)://(www.) to  https://
-        - traefik.http.middlewares.wwwtohttps.redirectregex.regex=^https?://(?:www\.)?(.+)
-        - traefik.http.middlewares.wwwtohttps.redirectregex.replacement=https://$${1}
-        - traefik.http.middlewares.wwwtohttps.redirectregex.permanent=true
+        - traefik.http.routers.www-catchall.rule=Host(`www.osparc.local`)
+        - traefik.http.routers.www-catchall.priority=100000
+        - traefik.http.routers.www-catchall.entrypoints=https,http
+        - traefik.http.routers.www-catchall.tls=true
+        - traefik.http.routers.www-catchall.middlewares=strip-www
+
         # Explicit www domain certificate
-        - traefik.http.routers.wwwsecure-catchall.tls.domains[0].main=www.${MACHINE_FQDN}
+        - traefik.http.routers.www-catchall.tls.domains[0].main=www.${MACHINE_FQDN} # Shouldnt this be all domains?>>
```

Comment:
@YuryHrytsuk please commentif you can :--)

Reply:
We do not rely on this after we moved towards manual certificate generation.

I believe this line can be removed. 

I cannot guarantee if it will cause some side-effects though

Reply:
ok I will remove it then :--)